src/sql/src/plan/statement/ddl.rs

```diff
@@ -367,10 +367,14 @@ pub fn plan_create_source(
 
             let encoding = get_encoding(format, envelope, with_options_original)?;
 
-            let mut connector = KafkaSourceConnector {
-                addrs: broker.parse()?,
+            let mut connector = KafkaSourceInstance {
+                // addrs: broker.parse()?,
```

Comment:
Vestigial?

Reply:
Yup, I double and triple checked since the moment I commented it out I began worrying I would forget to remove it and yet here it is. 🤣